examples/sap_hana_ha_simple/main.tf

```diff
@@ -23,11 +23,11 @@ module "sap_hana_ha" {
   source = "../../modules/sap_hana_ha"
```

Comment:
Should this be switched back? Could we target the GCS buckets instead to try and keep a single source of truth?

Reply:
We should update this to the registry URL which is where the latest (and other versions of the) module will live. That should be the source of truth moving forward. Right now its pointing to v0.5 which is the old stuff we cleared up.

I have a comment/bug below this to update it once a version is release.

Reply:
Sounds good.